packages/invoices/src/OutboundShipment/DetailView/modals/BatchesTable.tsx

```diff
@@ -239,6 +244,7 @@ export const BatchesTable: React.FC<BatchesTableProps> = ({
                 <ModalNumericInput
                   value={placeholderRow?.numberOfPacks ?? 0}
                   inputProps={placeholderInputProps}
+                  disabled={invoiceStatus !== InvoiceNodeStatus.New}
```

Comment:
wonder if that should be in the `ItemDetailsModal` 🤔  maybe should just pass in `placeholderEnabled`

Reply:
good point - that changes how we handle the invoice status. will raise another issue!